apps/wallet/tests/balanceChanges.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, test } from './fixtures';
+import { createWallet } from './utils/auth';
+import {
+    generateAddress,
+    generateKeypairFromMnemonic,
+    requestingSuiFromFaucet,
+} from './utils/localnet';
+
+const receivedAddressMnemonic = [
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+];
+
+const COIN_TO_SEND = 20;
+
+test('request SUI from local faucet', async ({ page, extensionUrl }) => {
+    await createWallet(page, extensionUrl);
+
+    await expect(page.getByTestId('coin-balance')).toHaveText('0SUI');
```

Comment:
I mentioned this in DMs as well, but I wouldn't assert specific balances, because it makes tests non-replayable with the same mnemonic and network. Instead, we can just get the text and validate that it changes.